### PR TITLE
lndclient: ChanInfo to include pending htlc values and direction

### PR DESCRIPTION
**Pull request by bhandras**
#### Pull Request Checklist

- [x] PR is opened against correct version branch.
~- [x] Version compatibility matrix in the README and minimal required version
      in `lnd_services.go` are updated.~

### CODE REVIEW

File: lightning_client.go [hunk 1]
```diff
@@ -1310,6 +1323,15 @@ type InvoiceHtlc struct {
 	ResolveTime time.Time
 }
 
+// PendingHtlc represents a HTLC that is currently pending on some channel.
+type PendingHtlc struct {
+	// Incoming is true if the HTLC is incoming.
+	Incoming bool
+
+	// Amount is the amount in millisatoshis that this HTLC represents.
+	Amount btcutil.Amount
```
Comment (guggero) [inline lightning_client.go:1332]: Should use `lnwire.MilliSatoshi` if it's in fact millisatoshis.

**Review by guggero — APPROVED** [review]
LGTM, just one comment about the HTLC amount :tada:

File: lightning_client.go (hunk 1 above)
Comment (bhandras) [reply lightning_client.go:1332]: It's in satoshis though

Comment (guggero) [reply lightning_client.go:1332]: Okay, then fixing the comment should be enough.

Comment (bhandras) [reply lightning_client.go:1332]: Sorry the comment was copy pasted. Fixed now.

**Pull request merged**